README.md

```diff
@@ -35,7 +35,6 @@ docker run --rm -it -e "PORT=8080" -p 8080:8080 sample-app
 A `pom.xml` file, or one of the other POM formats supported by the
 [Maven Polyglot plugin](https://github.com/takari/polyglot-maven) in the root
 directory is requried for these buildpacks to build your app.
```

Comment:
Maybe we can fix this as well while at it

```suggestion
directory is required for these buildpacks to build your app.
```

Reply:
Fixed in [a7a842a](https://github.com/heroku/buildpacks-jvm/pull/643/commits/a7a842a8ef235ed084609dcb67ac62bc891f9328)